zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -25,11 +25,15 @@
       <term name="note">wie Anm.</term>
       <term name="et-al">et al.</term>
       <term name="editor" form="short">Hg.</term>
+      <term name="translator" form="verb-short">übers. von</term>
+      <term name="editortranslator" form="verb-short">hg. und übers. von</term>
```

Comment:
You are right this works already here because of an exception I didn't know:

> If multiple variables are selected (separated by single spaces, see example below), each variable is independently rendered in the order specified, with one exception: when the selection consists of “editor” and “translator”, and when the contents of these two name variables is identical, then the contents of only one name variable is rendered. In addition, the “editortranslator” term is used if the cs:names element contains a cs:label element

http://docs.citationstyles.org/en/stable/specification.html#names